.npmrc

```diff
@@ -1 +1,2 @@
-registry=https://registry.npmjs.org
\ No newline at end of file
+registry=https://registry.npmjs.org
+package-lock=false
```

Comment:
Are you adding this in case someone uses `npm` locally?

Reply:
Yeah that is the main reason